app/assets/javascripts/govuk_publishing_components/lib/govspeak/barchart-enhancement.js

```diff
@@ -10,16 +10,26 @@ window.GOVUK = window.GOVUK || {};
   }
 
   BarchartEnhancement.prototype.init = function () {
-    // the not selectors are protection against initialising twice since that
+    // this is protection against initialising twice since that
     // causes quite a mess. The not .mc-chart is because it creates a second
     // .js-barchart-table element with .mc-chart and then the
     // .js-barchart-table-init is given when we initialise
-    var $barcharts = this.$element.find('.js-barchart-table:not(.mc-chart):not(.js-barchart-table-init)')
-    $barcharts.each(function () {
-      var $table = $(this)
-      $.magnaCharta($table, { toggleText: 'Change between chart and table' })
-      $table.addClass('js-barchart-table-init')
-    })
+    // we'd use :not selectors but for IE8, previously worked before jquery removal
```

Comment:
I _think_ I know what this comment is trying to say - still, I found it quite confusing to understand (but I appreciate that it's the result of layers of work over time).

I think it'd be worth splitting up and putting closer to the code that's doing what this comment is describing, which I think is the `if` statement on line 23.

Reply:
Ah, the perils of copying someone else's code into a repository. Yeah, this comment is a bit confusing, rewording.